src/components/Router/index.js

```diff
@@ -61,6 +63,10 @@ class CrossDeviceMobileRouter extends Component {
     this.state.socket.close()
   }
 
+  componentWillReceiveProps(nextProps) {
+    this.setState({i18n: setI18n(nextProps.options.language)})
```

Comment:
Minor: `createI18n` might more explicit, since it doesn't seem to be setting anything.